src/test/java/uk/nhs/prm/repo/ehrtransferservice/handlers/EhrCompleteHandlerTest.java

```diff
@@ -51,27 +51,27 @@ void setUp() {
     }
 
     @Test
-    public void shouldCallGp2gpMessengerServiceToSendPositiveAcknowledgement() throws Exception {
+    void shouldCallGp2gpMessengerServiceToSendPositiveAcknowledgement() throws Exception {
         ehrCompleteHandler.handleMessage(ehrCompleteEvent);
         verify(gp2gpMessengerService).sendEhrCompletePositiveAcknowledgement(ehrCompleteEvent, transfer);
     }
 
     @Test
-    public void shouldUpdateDbWithEhrTransferStatusWhenEhrRequestSentSuccessfully() throws Exception {
+    void shouldUpdateDbWithEhrTransferStatusWhenEhrRequestSentSuccessfully() throws Exception {
         ehrCompleteHandler.handleMessage(ehrCompleteEvent);
         verify(transferStore).handleEhrTransferStateUpdate(conversationId.toString(), "some-nems-message-id", "ACTION:EHR_TRANSFER_TO_REPO_COMPLETE", false);
     }
 
     @Test
-    public void shouldThrowErrorAndNotUpdateDbWhenFailsToSendPositiveAcknowledgement() throws Exception {
+    void shouldThrowErrorAndNotUpdateDbWhenFailsToSendPositiveAcknowledgement() throws Exception {
         doThrow(Exception.class).when(gp2gpMessengerService).sendEhrCompletePositiveAcknowledgement(ehrCompleteEvent, transfer);
 
         assertThrows(Exception.class, () -> ehrCompleteHandler.handleMessage(ehrCompleteEvent));
         verify(transferStore, never()).handleEhrTransferStateUpdate(conversationId.toString(),"some-nems-message-id","ACTION:EHR_TRANSFER_TO_REPO_COMPLETE", false);
     }
 
     @Test
-    public void shouldPublishTransferCompleteMessageToTransferCompleteTopic() throws Exception {
+    void shouldPublishTransferCompleteMessageToTransferCompleteTopic() throws Exception {
         var transferComplete = new TransferCompleteEvent("nemsEventLastUpdated", "some-ods-code", "SUSPENSION", "some-nems-message-id", "some-nhs-number");
 //        when(transferTrackerDbEntry.getNhsNumber()).thenReturn("some-nhs-number");
 //        when(transferTrackerDbEntry.getSourceGP()).thenReturn("some-ods-code");
```

Comment:
Should we remove these comments?

Reply:
Not a fan of that 'oops coming in backdoor' comment. It appears it only gets used on an EHR request timeout, I'll remove the comment for that.

Reply:
Also removed your suggested comments, good spot!